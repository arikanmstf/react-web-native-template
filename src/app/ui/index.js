export { default as Text } from './Text/Web';
export { default as Button } from './Button/Web';
export { default as View } from './View/Web';
