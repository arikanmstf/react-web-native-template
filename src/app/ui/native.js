export { default as Text } from './Text/Native';
export { default as Button } from './Button/Native';
export { default as View } from './View/Native';
export { default as Image } from './Image/Native';
