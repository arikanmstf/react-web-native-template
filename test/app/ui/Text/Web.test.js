import React from 'react';
import { shallow } from 'enzyme';
import { Text } from 'ui';

describe('src/app/ui/Text/Web.js', () => {
  it('Render: props: { children: DummyText }', () => {
    const wrapper = shallow(<Text>DummyText</Text>);
    expect(wrapper).toMatchSnapshot();
  });
});
