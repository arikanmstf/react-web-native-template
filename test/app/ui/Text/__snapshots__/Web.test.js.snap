// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/app/ui/Text/Web.js Render: props: { children: DummyText } 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Text>
    DummyText
  </Text>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "DummyText",
    },
    "ref": null,
    "rendered": "DummyText",
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "DummyText",
      },
      "ref": null,
      "rendered": "DummyText",
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
